### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by akramerin**
Added some buttons and some descriptive text on the landing page.

### CODE REVIEW

File: app/controllers/rentals_controller.rb [hunk 1]
```diff
@@ -6,24 +6,15 @@ def index
     @rentals = Rental.where(user_id: current_user.id)
   end
```
Comment (syboulette) [inline app/controllers/rentals_controller.rb]: def new
    @rental = Rental.new
    @book = Book.find(params[:book_id])
    authorize @rental
  end

File: db/schema.rb [hunk 2]
```diff
@@ -10,21 +10,22 @@
...
+ActiveRecord::Schema[7.0].define(version: 2023_02_24_184326) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "books", force: :cascade do |t|
     t.string "title"
     t.string "author"
     t.string "genre"
-    t.date "date_of_publication"
+    t.integer "date_of_publication"
```
Comment (syboulette) [inline db/schema.rb]: That does not work, please keep date of publication as a date :)

**Review by syboulette — CHANGES_REQUESTED** [review]

File: app/controllers/rentals_controller.rb (hunk 1 above)
Comment (akramerin) [reply app/controllers/rentals_controller.rb]: check! thanks

File: db/schema.rb (hunk 2 above)
Comment (akramerin) [reply db/schema.rb]: check!

File: app/controllers/rentals_controller.rb [hunk 3]
```diff
@@ -6,29 +6,27 @@ def index
...
   def create
     @rental = Rental.new(rental_params)
-    @rental.user = current_user
-    @rental.book = Book.find(params[:book_id])
-
     authorize @rental
-
+    @rental.user = current_user
     if @rental.save!
-      redirect_to root_path
```
Comment (akramerin) [inline app/controllers/rentals_controller.rb]: please just don't accept the changes in the rental file -  i did not make them, I think they are a merging error

**Review by syboulette — APPROVED** [review]

**Pull request merged**